kubernetes/loculus/values.yaml

```diff
@@ -25,6 +25,11 @@ dataUseTerms:
   urls:
     open: https://#TODO-MVP/open
     restricted: https://#TODO-MVP/restricted
+sequenceFlagging:
+  github:
+    organization: pathoplexus
+    repository: curation_reports
+    issueTemplate: sequence-metadata-issue.md
```

Comment:
By default, this should be disabled (we can turn it on for the previews).

Reply:
makes sense.

Reply:
disabled it here: https://github.com/loculus-project/loculus/pull/3681/commits/422d568970e65243593483d4e9011ea6d0f30b8a

Reply:
We should enable for previews